pom.xml

```diff
@@ -4,11 +4,12 @@
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>dev.cdevents</groupId>
-	<artifactId>cdevents-sdk-java</artifactId>
+	<artifactId>cdevents-sdk-java-parent</artifactId>

```

Comment:
I guess maven requires a parent artifact to exists? 
If not, it would be nice to have just the one artifact for the SDK, I don't think we need to publish the generator to maven really unless we think it would be useful for others?

Reply:
Only the parent pom and the sdk library (pom, jar, -sources.jar, -javadoc.jar) will be published. the generator is already configured to skip publication.